reservacionVuelo/android/app/build.gradle

```diff
@@ -1,6 +1,7 @@
-apply plugin: "com.android.application"
+def var = apply plugin: "com.android.application"
+var
 project.ext.vectoricons = [
-    iconFontNames: [ 'MaterialIcons.ttf', 'EvilIcons.ttf' ] // Name of the font files you want to copy
+    iconFontNames: [ 'MaterialIcons.ttf', 'EvilIcons.ttf', 'FontAwesome.ttf','MaterialCommunityIcons.ttf'] // Name of the font files you want to copy
 ]
 apply from: "../../node_modules/react-native-vector-icons/fonts.gradle"
 
```

Comment:
Apparently you have a conflict in this file with the main git branch. If you can, fix that conflict before you do merge.

![imagen](https://user-images.githubusercontent.com/12243331/114255659-435a8e00-997a-11eb-9610-49c766735110.png)

Reply:
Also the same as Yil, hahaha xD
So that later that will not cause problems for them.